constants/appointment_made_by.go

```diff
@@ -0,0 +1,31 @@
+package constants
+
+// AppointmentMadeBy Enum Type
+type AppointmentMadeBy int
+
+// Enumeration containing all possible practitioner roles
```

Comment:
This contains all the possible appointment made by options not practitioner roles